src/Sylius/Component/Translation/Factory/TranslatableFactoryInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Translation\Factory;
+
+use Sylius\Component\Resource\Factory\FactoryInterface;
+
+/**
+ * Interface for factories creating translatable resources.
+ *
+ * @author Paweł Jędrzejewski <pawel@sylius.org>
+ */
+interface TranslatableFactoryInterface extends FactoryInterface
```

Comment:
This interface is a marker right now and I think we should keep it, cause I am thinking about introducing some methods like ``createForLocale`` etc.

As for PHPDoc - Yes, when we rework translations we should add it here, but not AbstractTranslatable but probably TranslatableInterface. :) WDYT? (about both points)